README.md

```diff
@@ -1 +1,2 @@
-# Onliner
\ No newline at end of file
+# Onliner
+Для запуска необходимо скачать ChromeDriver необходимой версии https://chromedriver.chromium.org/downloads и поместить в дирректорию src/test/resources
```

Comment:
Is this info still needed? 
Please explain TAF technologies and features in the README file

Reply:
fixed